demo/edge-attachmediastream.js

```diff
@@ -3,7 +3,7 @@
  * And Edge current does not support re-negotiation for now in the SDK.
  */
 (function () {
-  if (window.webrtcDetectedBrowser === 'edge') {
```

Comment:
The release was to be able to handle the older Edge and also handle the latest Edge if possible (unless we have to use legacy API depending on time constraints).